### PR TITLE
legacy: removal of inveniocfg command

### PR DESCRIPTION
**Pull request by jirikuncar**
* INCOMPATIBLE Removes deprecated `inveniocfg` command line interface.

Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

--
cc @dset0x

### CODE REVIEW

File: invenio/base/scripts/apache.py [hunk 1]
```diff
@@ -19,11 +19,57 @@
...
 
+import os
+import socket
+
 from invenio.ext.script import Manager, change_command_name
 
 manager = Manager(usage="Perform Apache operations.")
 
 
+def _detect_ip_address():
```
Comment (dset0x) [inline invenio/base/scripts/apache.py]: Take a look at lines 82 and 117 in this file where this and the next function are imported from `invenio.legacy.inveniocfg`.

**Comment by dset0x** [comment]
Should `invenio/testsuite/deprecated_test_inveniocfg.py` also be removed along with this?

File: invenio/base/scripts/apache.py (hunk 1 above)
Comment (jirikuncar) [inline invenio/base/scripts/apache.py]: thanks

**Comment by jirikuncar** [comment]
> Should `invenio/testsuite/deprecated_test_inveniocfg.py` also be removed along with this?

:+1:

**Comment by jirikuncar** [comment]
@dset0x thank you for the review. I have mentioned you in the commit message.

**Comment by dset0x** [comment]
@jirikuncar No problem. greppable code is useful with python:

`ack --type=python 'import.* inveniocfg|inveniocfg.* import '`

pylint also helps.

**Comment by jirikuncar** [comment]
PS: @dset0x it happened because I just didn't save the file before commit :stuck_out_tongue_winking_eye:

**Pull request merged**